### DISCUSSION TITLE
Transmission 3.0 brute force protection being triggered and causing transmission to stop

### DISCUSSION THREAD

**Discussion by Nurgus** [haugene/docker-transmission-openvpn#2355] [Q&A]
I added the script to install transmission 3 as I was migrating an existing transmission setup into this docker and didn't want to go through reverting back to 2.x (having gone through some pain upgrading to 3 already..!)

Everything was working great until I noticed the RPC was getting locked out due to 100 failed authentications - a feature of transmission 3 and not something anyone should be turning off imo.

I eventually found a solution - add the RPC username and password via environment variables to the Docker run so that they get added to transmission-credentials.txt

I take it then that the docker has a script that connects to the RPC for some reason? Is there a solution that doesn't require me to have the credentials in the docker's environment variables?

**Comment by edgd1er** [comment]
Some variables may be defined through secrets:
https://github.com/haugene/docker-transmission-openvpn/issues/534

**Reply by Nurgus** [reply]
Thanks, will have a look at that. 

I don't understand why there's a need for something in the docker to connect to my transmission's rpc ?

**Reply by pkishino** [reply]
I think the only thing that would do that is possibly port forwarding script depending on your provider. Apart from that I’m not aware of anything using rpc
